cmd/generate-config/config/config-openapi-spec.json

```diff
@@ -73,9 +73,17 @@
     "ingress": {
       "type": "object",
       "required": [
+        "expose",
         "routeAdmissionPolicy"
       ],
       "properties": {
+        "expose": {
```

Comment:
How about listenAddress? It also allows NIC names, but always translates to IP addresses.

Reply:
What is the field called in the kube API where these values end up being copied?

Reply:
If we call it `listenAddresses` how confusing will it be that it takes an interface, too? Does `listenInterfaces` have the connotation that it could be a NIC name or IP?

Reply:
Suggested `listenaddress` because after all everything gets translated into IP addresses, and users can see it in the service itself. It is true that it does not accommodate NIC names very good though. I see `listenInterfaces` with a similar issue but on the opposite direction. What do you think of `endpoints`? That would serve for anything that accepts network connections.